locales-id-ID.xml

```diff
@@ -69,13 +69,13 @@
     <term name="page-range-delimiter">–</term>
 
     <!-- ORDINALS -->
-    <term name="ordinal"></term>
-    <term name="ordinal-01"></term>
-    <term name="ordinal-02"></term>
-    <term name="ordinal-03"></term>
-    <term name="ordinal-11"></term>
-    <term name="ordinal-12"></term>
-    <term name="ordinal-13"></term>
+    <term name="ordinal">bilangan bertingkat</term>
```

Comment:
No, it is right. If it doesn't support prefix, it should be left as the original for time being. Indonesian as Malay language use prefix to denote ordinal rather than suffix as English does.

Perhaps, someday this will be supported as Malay, Filipino and Chinese used prefix to denote ordinal numbers.